packages/server/src/ApolloServer.ts

```diff
@@ -270,7 +273,12 @@ export class ApolloServer<TContext extends BaseContext = BaseContext> {
     // maintained Keyv packages or our own Keyv store `LRUCacheStore`).
     // TODO(AS4): warn users and provide better documentation around providing
     // an appropriate Keyv.
-    const cache = config.cache ?? new Keyv();
+    //
+    // FIXME: @glasser do we want to just use the
```

Comment:
This is what we did in AS3 and seems fine? I guess we were maybe getting rid of InMemoryLRUCache before?